sixMans/data_example.json

```diff
@@ -64,15 +64,17 @@
         },
         "scores": [
             {
-                "queue": "queue1",
-                "playerId": "id",
+                "game": "gameId",
+                "queue": "queueId",
+                "player": "playerId",
```

Comment:
That would be a significant departure from the current setup. I like that idea but it would require a way to alter the existing data to fit the new format, which is the issue with any sort of major data change.

Reply:
Agreed. I wonder if we could retrospectively assign IDs to old scores and from there, run a script to refactor the existing json.

It might be worth adding the `blue`, `orange`, and `winner` fields, so its easier to make this transition in the future? Just an idea.

Reply:
We'll consider making changes like that in a future update but for now that is outside the scope of this cleanup

Reply:
👍